packages/firebase_analytics/firebase_analytics/example/pubspec.yaml

```diff
@@ -6,13 +6,14 @@ environment:
   flutter: ">=1.12.13+hotfix.5"
 
 dependencies:
-  e2e: ^0.6.1
```

Comment:
Is there a reason for this change?

Reply:
As e2e is now deprecated and replaced by `integration_test` I hoped that I would gain null-safety, but I then realized that the `intergation_test` null-safety migration did not make this stable release of Flutter ...

Reply:
Yes, it seems `integration_test` and `flutter_driver` are holding many packages back from null safety. In some cases, developers are foregoing the dependency to enable their users to migrate to null safety.